src/error.rs

```diff
@@ -0,0 +1,10 @@
+//! Module to export tide_core errors
+
+use core::pin::Pin;
+use futures::future::Future;
+
+pub use tide_core::error::{
+    EndpointResult, Error, ResponseExt, ResultDynErrExt, ResultExt, StringError,
+};
+
+pub(crate) type BoxTryFuture<T> = Pin<Box<dyn Future<Output = EndpointResult<T>> + Send + 'static>>;
```

Comment:
This file doesn't seems unrelated to documentation; was this perhaps checked in accidentally?

Reply:
I'm sorry. I missed my self review. I fixed it!